tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -432,7 +454,22 @@ public function removeUserFromGroup(string $groupId, string $userId, ?string $by
 	 * @throws GuzzleException
 	 */
 	public function theUserDeletesAUserUsingTheGraphAPI(string $byUser, string $user): void {
-		$this->adminDeletesUserUsingTheGraphApi($user, $byUser);
+		$userId = $this->featureContext->getUserIdByUserName($user);
+		$this->adminDeletesUserByUserIdUsingTheGraphApi($userId, $byUser);
```

Comment:
We can use username to delete user, so no need to get the user ID
```suggestion
		$this->adminDeletesUserUsingTheGraphApi($user, $byUser);
```

Reply:
@saw-jan answer of all your suggested changes related to this `i asked Viktor which method should we use he told me to use user id to delete the user. So i implemented deletion of user using user-id`